src/data/mystery-encounters/mystery-encounter.ts

```diff
@@ -90,6 +111,11 @@ export default interface IMysteryEncounter {
    * You probably shouldn't do anything with this unless you have a very specific need
    */
   introVisuals?: MysteryEncounterIntroVisuals;
+  /**
+   * Used for keeping RNG consistent on session resets, but increments when cycling through multiple "Encounters" on the same wave
+   * You should never need to modify this
+   */
+  seedOffset?: any;
```

Comment:
If you never have to modify this, is it possible to make it `readonly`?

Reply:
ended up moving to the class and making private, since it's necessary to increment